<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteTasksDialog</class>
 <widget class="QDialog" name="RemoteTasksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Remote Tasks</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Remote service URL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>&lt;unknown&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTreeWidget" name="tasksTreeWidget">
     <column>
      <property name="text">
       <string>Task ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Submitted</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>State</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Result</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="refreshPushButton">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fetchPushButton">
       <property name="text">
        <string>Fetch data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removePushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Remove task data</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemoteTasksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemoteTasksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
